src/App.jsx

```diff
@@ -14,8 +34,23 @@ export default function App() {
           <Instructions />
         </InstructionsButton>
       </header>
+      <p>Current Player: {calculateCurrentPlayerMark(currentPlayerId)}</p>
+
       <div className="App-body">
-        <Board data={boardData} />
+        <div>
+          {isAlertVisible && (
+            <Alert
+              message="Move already made. Please choose an empty cell."
```

Comment:
This `message` prop might be a pre-optimization. This is the only place that `Alert` is being used.  I would move the message text into Alert until you actually need it to be a prop.

Reply:
oh true! good point.